contracts/tokenomics/generator/tests/integration.rs

```diff
@@ -1019,14 +969,13 @@ fn instantiate_generator(mut app: &mut App, astro_token_instance: &Addr) -> Addr
 
     let amount = Uint128::new(63072000_000000);
 
-    let msg = Cw20ExecuteMsg::IncreaseAllowance {
-        spender: vesting_instance.to_string(),
-        amount,
-        expires: None,
-    };
-
-    app.execute_contract(owner.clone(), astro_token_instance.clone(), &msg, &[])
-        .unwrap();
+    allow_tokens(
```

Comment:
Do we need allowance here?

Reply:
yep, we are increase allowance from the owner token for registered Vesting contracts.